Documentation/gettingstarted/encryption-wireguard.rst

```diff
@@ -21,6 +21,10 @@ in the Kubernetes ``CiliumNode`` custom resource object. Each node's public key
 is then used by other nodes to decrypt and encrypt traffic from and to
 Cilium-managed endpoints running on that node.
 
+Packets destined to the same node they were sent out of are not encrypted.
+This is a intended behavior as it doesn't provide any benefits because the
```

Comment:
```suggestion
This is an intended behavior as it would not provide any benefits because the
```

Reply:
I copy pasted this sentence from the ipsec page but I'm happy to update both instances.